docs/developer-guide/ksqldb-reference/select-pull-query.md

```diff
@@ -36,12 +36,40 @@ Execute a pull query by sending an HTTP request to the ksqlDB REST API, and
 the API responds with a single response.  
 
 The WHERE clause must contain a value for each primary-key column to retrieve and may
-optionally include bounds on `WINDOWSTART` and `WINDOWEND` if the materialized table is windowed.
+optionally include bounds on `WINDOWSTART` and `WINDOWEND` if the materialized view is windowed.
 For more information, see 
 [Time and Windows in ksqlDB](../../concepts/time-and-windows-in-ksqldb-queries.md).
 
 Example
 -------
+You can issue a pull query against a derived table that was created by using 
+the [CREATE TABLE AS SELECT](../../ksqldb-reference/create-table-as-select) 
+statement. For example, you can create a table named `GRADES` by using a [CREATE TABLE](../../ksqldb-reference/create-table) 

```

Comment:
I think we should have two sections: `Queryable views` and `WHERE clause guidelines`.  They could be before the examples and just summarize each.

`Queryable views` would state Almog's assertion like:
Any table created with `AS SELECT` can be queried.  Some examples include:
- Tables created by querying another table
- Tables created with a group by clause
- Tables created by joining other tables.

`WHERE clause guidelines` would state that by default, only key lookups are enabled.  These have the following requirements:
- Key column(s) must use an equality comparison to a literal (e.g. `KEY = 'abc'`)
- On windowed tables, `WINDOWSTART` and `WINDOWEND` can be optionally compared to literals.

To enable table scans, you set `ksql.query.pull.table.scan.enabled=true`.  Once they're are enabled, additional expressions are allowed:
- Key column(s) using range comparisons to literals
- Non key columns to be used alone, without key references
- Columns to be compared to other columns
- References to subsets of columns from a multi-column key

Optionally, these can be worked into the examples, but I think it would be useful to summarize.  Also should probably add some more examples that hit some of the above, especially table scans.